wordpress-mobile/WordPress-Android
# Issue/13332 hide certain my site items for p2

Fixes #13332

This PR hides Activity Log, Plans, and Themes from P2 sites.

We are not hiding wp-admin yet.

To test:
- Switch to P2 and navigate to My Site.
- Notice Activity Log, Plans, and Themes are not visible.
- Switch to a non-P2 site.
- Confirm that Activity Log, Plans, and Themes are visible when allowed.

PR submission checklist:
